spring-boot/src/main/java/org/springframework/boot/context/embedded/Compression.java

```diff
@@ -31,12 +31,17 @@
 	private boolean enabled = false;
 
 	/**
-	 * Comma-separated list of MIME types that should be compressed.
+	 * MIME types that should be compressed.
```

Comment:
The javadoc on this field is used in the [configuraton metadata](http://docs.spring.io/spring-boot/docs/current-SNAPSHOT/reference/htmlsingle/#configuration-metadata) that's automatically collected as part of the build. While the field itself isn't a comma-separated list, that how we chose to describe it as that (typically) how it'll be configured in `application.properties`.